src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,73 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.jsx';
+
+
+const ChatEntry = (props) => {
+  const likeButtonClicked = () => {
+    props.onLikeToggle(props.id);
+  };
+
+
+  const getSenderType = () => {
+    if (!props.twoPeople) {
+      return '';
+    } else if (props.twoPeople.length === 2) {
+      if (props.sender === props.twoPeople[0]) {
+        return 'chat-entry local';
+      } else if (props.sender === props.twoPeople[1]) {
+        return 'chat-entry remote';
+      }
+    } else if (props.twoPeople.length === 1) {
+      if (props.sender === props.twoPeople[0]) {
+        return 'chat-entry local';
+      }
+    }
+    return '';
+  };
+
+
+  const setColor = () => {
+    if (!props.colorChoice || !props.twoPeople) return '';
+
+    if (props.twoPeople.length === 2) {
+      if (props.sender === props.twoPeople[0]) {
+        return props.colorChoice.vColor;
+      } else if (props.sender === props.twoPeople[1]) {
+        return props.colorChoice.eColor;
+      }
+    } else if (props.twoPeople.length === 1) {
+      return props.colorChoice.vColor;
+    }
+
+    return '';
+  };
+
 
-const ChatEntry = () => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={getSenderType()}>
```

Comment:
Another option could be to have an interpolated string here that always holds `chat-entry` and use a placeholder where we pass only the `remote` or `local` class name (so we don't repeat `chat-entry` anywhere):
```javascript
const entryClassName = (props.sender === 'Vladimir') ? 'local' : 'remote'; // Or however you are handling multiple users in the chat
...
<div className={`chat-entry ${entryClassName}`}>
```